notebooks/03.09-Pivot-Tables.ipynb

```diff
@@ -49,9 +49,7 @@
   {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    titanic.pivot_table('survived', ['sex', age], [fare, 'class'])
</pre><p>this is line 3 comment</p>
<br/>

 _Reply via <a href='http://127.0.0.1:8000/amit1rrr/PythonDataScienceHandbook/pull/29/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>